tldw_Server_API/app/services/admin_system_service.py

```
@@ -699,3 +747,276 @@ async def list_system_logs(
+            is_expired = exp_dt < datetime.now(timezone.utc)
+
+        return {
+            "valid": True,
+            "header": header,
+            "payload": {k: v for k, v in payload.items() if k not in ("password", "secret")},
+            "expired": is_expired,
+            "expires_at": datetime.fromtimestamp(exp, tz=timezone.utc).isoformat() if exp else None,
+            "issuer": payload.get("iss"),
+            "subject": payload.get("sub"),
+        }
+    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)